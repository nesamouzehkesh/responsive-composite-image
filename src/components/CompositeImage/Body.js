import React, {Component} from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const StyledImage = styled.img`
  width: 100%;
  height: 100%;
`;

function Body({src}) {
  return <StyledImage src={src} />;
}

Body.propTypes = {
  src: PropTypes.string.isRequired,
};

export default Body;
