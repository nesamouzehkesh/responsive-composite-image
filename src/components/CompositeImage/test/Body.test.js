import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Body from '../Body';

describe ('Test CompositeImage Body component', () => {
  it ('renders Body component normally and check with snapshot', () => {
    const element = renderer.create (<Body src="test.png" />).toJSON ();

    expect (element).toMatchSnapshot ();
  });
});
