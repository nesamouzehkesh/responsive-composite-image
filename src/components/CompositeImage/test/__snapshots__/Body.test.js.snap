// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CompositeImage Body component renders Body component normally and check with snapshot 1`] = `
<img
  className="sc-bdVaJa ehJlOl"
  src="test.png"
/>
`;
